UI/window-basic-main-outputs.cpp

```diff
@@ -850,7 +850,19 @@ void SimpleOutput::Update()
 					   audioSettings);
 
 	if (!enforceBitrate) {
-		blog(LOG_INFO, "User is ignoring service bitrate limits.");
+		int MaxVBitrate, MaxABitrate;
+		obs_service_get_max_bitrate(main->GetService(), &MaxVBitrate,
+					    &MaxABitrate);
+
+		std::string videoBitrateStr =
+			MaxVBitrate > 0 ? std::to_string(MaxVBitrate) : "None";
+		std::string audioBitrateStr =
+			MaxABitrate > 0 ? std::to_string(MaxABitrate) : "None";
+
+		blog(LOG_INFO,
+		     "User is ignoring service bitrate limits.\nService Recommendations:\n\tvideo bitrate: %s\n\taudio bitrate: %s",
```

Comment:
This string and the other string in the other part of this PR it can be split into like two or three segments, e.g.

```cpp
log(LOG_INFO,
		     "User is ignoring service bitrate limits.\n"
		     "Service Recommendations:\n\tvideo bitrate: %s\n"
		     "\taudio bitrate: %s",
		     videoBitrateStr.c_str(), audioBitrateStr.c_str());
```

(clang-format will probably figure out a better formatting but this is just an example)

Overall doing this just makes the code a bit more clean.

Reply:
Updated!